src/main.cc

```diff
@@ -51,7 +51,8 @@ void Serialization(std::ostream *stream,
     *stream << std::flush;
 }
 
-int ProcessAST(const Config& config, std::unique_ptr<std::istream>& in, std::unique_ptr<std::ostream>& out) {
```

Comment:
Can we do that **just** before the release? There are several things we are working on which might be messed up.